source/containerimage/pull.go

```diff
@@ -152,6 +152,7 @@ func (p *puller) CacheKey(ctx context.Context, g session.Group, index int) (cach
 					labels = make(map[string]string)
 				}
 				maps.Copy(labels, estargz.SnapshotLabels(p.manifest.Ref, p.manifest.Descriptors, i))
+				labels["containerd.io/snapshot/image-ref"] = p.manifest.Ref
```

Comment:
Same as the above. `containerd.io/snapshot/image-ref` should be explained somewhere if it's specific to overlaybd.

Reply:
we have fixed this label.

Could you re-run the test?
It should be some environmental reason that caused the test to fail